src/govuk/common.unit.test.mjs

```diff
@@ -216,4 +217,32 @@ describe('Common JS utilities', () => {
       })
     })
   })
+
+  describe('closestAttributeValue', () => {
+    it('returns the value of the attribute if on the element', () => {
+      const $element = createElement('div', { lang: 'en-GB' })
+
+      expect(closestAttributeValue($element, 'lang')).toEqual('en-GB')
+    })
+
+    it('returns the value of the closest parent with the attribute if it exists', () => {
+      const $greatGrandParent = createElement('div', { lang: 'cy-GB' }) // To check that we take the value up
+      const $grandparent = createElement('div', { lang: 'en-GB' })
+      const $parent = createElement('div') // To check that we walk up the tree
+      const $element = createElement('div')
+      $greatGrandParent.appendChild($grandparent)
+      $grandparent.appendChild($parent)
+      $parent.appendChild($element)
```

Comment:
I like the `createElement` helper function, but I do wonder if this would be easier to follow if this was written out in HTML rather than constructed by appending individual children. The naming does help, but it's a _little_ hard to follow.

Reply:
Fair point, I'll see if I can rewrite this to work from an HTML snippet.